src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working.");
```

Comment:
Write more informative messages in methods. The message should indicate what type of `Machine` is working right now. For example: `Bulldozer started its work.`